docs/doxygen-user/installMultiUser.dox

```diff
@@ -0,0 +1,31 @@
+/*! \page install_multiuser_page Setting Up Multi-user Environment
+
+\section multiuser_install Multi-user Installation
+
+Autopsy can be setup to work in an environment where multiple users on different computers can have the same case open at the same time.  To setup this type of environment, you will need to configure additional (free and open source) network-based services. 
+
+\subsection multiuser_install_services Network-based Services
+
+You will need the following that all Autopsy clients can access:
+- Centralized storage that all clients running Autopsy have access to. The central storage should be either mounted at the same Windows drive letter or UNC paths should be used everywhere. All clients need to be able to access data using the same path.
+- A central PostgreSQL database.  A database will be created for each case and will be stored on the local drive of the database server. Installation and configuration is explained in \ref install_postgresql.
+- A central Solr text index. A Solr core will be created for each case and will be stored in the case folder (not on the local drive of the Solr server). We recommend using Bitnami Solr. This is explained in \ref install_solr.
+- An ActiveMQ messaging server to allow the various clients to communicate with each other.  This service has minimal storage requirements. This is explained in \ref install_activemq.
+
+When you setup the above services, write down the addresses, user names, and passwords or each so that you can configure each of the client systems afterwards. 
+
+We recommend using at least 2 dedicated computers for this additional infrastructure.  Spreading the services out across several machines can improve throughput. 
+If possible, place Solr on a machine by itself, as it is the largest RAM and CPU utilizer among the servers. 
+
+Ensure that the central storage and PostgreSQL servers are regularly backed up. 
+
+\subsection multiuser_install_clients Autopsy Clients
+
+Once the infrastructure is in place, you will need to configure Autopsy to use them. 
+- Install Autopsy on each client system as normal using the steps from \ref installation_page.
+- Start Autopsy and open the multi-user options panel from Tools -> Options -> Multi-user.  As shown in the screenshot below, you can then enter all of the address and authentication information for the network-based services. Note that in order to create or open Multi-user cases, "Enable Multi-user cases" must be checked and the settings below must be correct.
```

Comment:
I think we're using the format "Tools", Options, "Multi-user" pretty much everywhere now.